Ginger/GingerCoreNET/Run/GingerExecutionEngine.cs

```diff
@@ -4820,6 +4840,29 @@ private void NotifyBusinessFlowEnd(BusinessFlow businessFlow)
 
         }
 
+        private void NotifyBusinessFlowSkippedEnd(BusinessFlow businessFlow)
+        {
+            try
+            {
+                Reporter.ToStatus(eStatusMsgKey.StaticStatusProcess, null, "Updating status for pending activities...");
+                uint eventTime = RunListenerBase.GetEventTime();
+                businessFlow.EndTimeStamp = DateTime.UtcNow;
+                foreach (RunListenerBase runnerListener in mRunListeners)
+                {
+                    runnerListener.BusinessFlowSkippedEnd(eventTime, CurrentBusinessFlow);
```

Comment:
i didn't saw any call to "NotifyBusinessFlowSkippedEnd", why? also BF's can be skipped

Reply:
BF is handling/capturing the "skipped" event in the existing/original code already